src/api/app/views/webui/request/beta_show_tabs/_overview.html.haml

```diff
@@ -1 +1,61 @@
-Overview
+.row
+  .col-md-8#overview
+    #description-text
+      - if bs_request.description.present?
+        = render partial: 'webui/shared/collapsible_text', locals: { text: bs_request.description }
+      - else
+        %i No description set
+
+  .col-md-4#side-links
+    %ul.list-unstyled
+      - unless package_maintainers.empty?
+        %li
+          %i.fas.fa-info-circle.text-info
+          = render partial: 'maintainer_link', locals: { maintainers: package_maintainers }
+      - if bs_request.superseding.present?
+        %li
+          %i.fas.fa-plus-circle.text-primary
+          Supersedes
+          - bs_request.superseding.each do |supersed|
+            = link_to supersed['number'], number: supersed['number']
+      - if bs_request.accept_at.present?
+        %li
+          %i.fas.fa-exclamation-circle.text-danger
+          - if BsRequest::FINAL_REQUEST_STATES.include?(bs_request.state)
+            Auto-accept was set to
+            %span.fuzzy-time{ title: "#{l bs_request.accept_at}" }
+            = succeed '.' do
+              = l bs_request.accept_at, format: :only_date
+          - elsif bs_request.accept_at.past?
+            This request will be automatically accepted when it enters the 'new' state.
+          - else
+            This request will be automatically accepted in #{time_ago_in_words(bs_request.accept_at)}.
+      - bs_request.history_elements_accepted_or_declined.each do |state, histories|
```

Comment:
Hm this is #12858 too right?

Reply:
Yes, there's a bit of duplication... We need to talk

Reply:
This piece of code has been removed. I'm now only relying on review objects